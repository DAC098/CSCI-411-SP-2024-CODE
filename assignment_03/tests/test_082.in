6 10
356 417 628 746 814 868
8353
7771
4690
511
2760
2999
2938
7316
7067
2148
