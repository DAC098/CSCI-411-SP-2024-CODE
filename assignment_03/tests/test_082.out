0 1 0 3 7 0 
0 1 4 3 0 3 
2 0 1 1 0 3 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 0 0 0 0 
0 0 2 0 1 1 
3 0 2 0 4 2 
0 1 1 0 1 6 
0 0 0 0 0 0 
